.github/workflows/test-composite-actions.yaml

```
@@ -146,8 +146,7 @@ jobs:
       matrix:
         container:
           - ros:humble
-          - ghcr.io/autowarefoundation/autoware:latest-autoware-universe
-
+          - ghcr.io/autowarefoundation/autoware:latest-universe-devel-amd64
```

I manually ran the workflow https://github.com/autowarefoundation/autoware/actions/runs/10720030706. So that  the `autoware:latest-universe-devel` exists now.
https://github.com/autowarefoundation/autoware/pkgs/container/autoware/268898646?tag=latest-universe-devel

```suggestion
          - ghcr.io/autowarefoundation/autoware:latest-universe-devel
```